index.html

```diff
@@ -1,60 +1,85 @@
-
-        * A
-        * B
-
-    -->
-
-    <!--
-
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
-
-    -->
-
-    <h1>Assignment 01 - To Be Completed</h1>
-  </body>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Resume</title>
+</head>
+
+<body>
+  <header>
+    <h1>Online Resume</h1>
+    <img src="./resume assignment second image .jpeg" width="160" alt="Image">
+  </header>
+  <nav>
+    <ul>
+      <li><a href="AboutMe.html">About Me</a></li>
+      <li><a href="Skills.html">Skills</a></li>
+      <li><a href="ContactInformation.html">Contact Information</a></li>
+      <li><a href="Education.html">Education</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h2>Skills</h2>
+    <ul>
+      <li>Multitasker</li>
+      <li>Adaptive to collaboration</li>
+      <li>Great at providing customer service</li>
+      <li>Sales training</li>
+      <li>Effective and leading in teamwork</li>
+    </ul>
+
+    <h3>About me</h3>
+    <p>Organized and dependable student successful at managing multiple priorities with a positive attitude. Willingness
+      to take on added responsibilities to meet team goals. Determined and driven to provide customers with the best
+      experience possible.</p>
+
+    <h4>Contact Information</h4>
+    <section>
+      <p>Email: abdullahshibib@gmail.com</p>
+      <p>Phone number: 613-668-0982</p>
+    </section>
+
+    <h5>Work experience</h5>
+    <ul>
+      <li>Sales Associate, Hudson's Bay | Ottawa, ON
+        Organized racks and shelves to maintain store visual appeal, engage customers, and promote specific merchandise.
+        Provided positive first impressions to welcome existing, new, and potential customers.</li>
+    </ul>
+  </main>
+
+  <h2>Education</h2>
+  <section>
+    <p>High School Diploma - Woodroffe High School, 2410 Georgina Dr, Ottawa, ON K2B 7M8</p>
+    <p>Carleton University, Information Resource Management, 1125 Colonel By Dr, Ottawa, ON K1S 5B6</p>
+  </section>
+
+  <footer>
+    <h2>Site footer Information</h2>
+     <img src="./resume assignment second image .jpeg" width="160" alt="Image">
+    <nav>
+      <ul>
+        <li><a href="AboutMe.html">About Me</a></li>
+        <li><a href="Skills.html">Skills</a></li>
+        <li><a href="ContactInformation.html">Contact Information</a></li>
+        <li><a href="Education.html">Education</a></li>
+      </ul>
+      <form action="#">
+        <fieldset>
+          <legend>Enter your information</legend>
+          <label for="name">Name:</label>
+          <input required="required" type="text" id="name" placeholder="Abaan Noman" autocomplete="given-name">
+          <br>
+          <label for="email">Email:</label>
+          <input required="required" type="email" id="email" placeholder="abaannoman@gmail.com">
+        </fieldset>
+        <button type="submit">Submit your message here</button>
+      </form>
+    </nav>
+    <h5> © 2023 </h5>
```

Comment:
This can be a header - but if it is a header it should be wrapped in a section tag. 

It's also set as an h5, but there isn't an h4 in your footer. 

Please review heading rules in the class notes.